docs/tutorial/part-three/index.md

```diff
@@ -1,57 +1,57 @@
 ---
-title: Creating Nested Layout Components
+title: Tworzenie komponentów zagnieżdżonych układów
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Welcome to part three!
+Witaj w części trzeciej!
 
-## What's in this tutorial?
+## Czego nauczysz się w tym poradniku?
 
-In this part, you'll learn about Gatsby plugins and creating "layout" components.
+W tej części dowiesz się o wtyczkach Gatsby i tworzeniu komponentów „układu”.
 
-Gatsby plugins are JavaScript packages that help add functionality to a Gatsby site. Gatsby is designed to be extensible, which means plugins are able to extend and modify just about everything Gatsby does.
+Wtyczki Gatsby to paczki JavaScript, które pomagają dodać różne funkcjonalności do strony. Gatsby zaprojektowano tak aby był jak najbardziej elastyczny, co oznacza, że wtyczki mogą rozszerzać i modyfikować niemal wszystko, co robi.
 
-Layout components are for sections of your site that you want to share across multiple pages. For example, sites will commonly have a layout component with a shared header and footer. Other common things to add to layouts are a sidebar and/or navigation menu. On this page for example, the header at the top is part of gatsbyjs.org's layout component.
+Komponenty układu budują sekcje witryny, które chcesz udostępnić na wielu stronach. Na przykład, witryny zwykle zawierają komponent układu z takim samym nagłówkiem i stopką. Inne typowe elementy układów to pasek boczny i / lub nawigacja. Na przykład na tej stronie nagłówek u góry jest częścią komponentu układu gatsbyjs.org.
 
-Let's dive into part three.
+Sprawdźmy teraz część trzecią poradnika.
 
-## Using plugins
+## Używanie wtyczek
 
-You’re probably familiar with the idea of plugins. Many software systems support adding custom plugins to add new functionality or even modify the core workings of the software. Gatsby plugins work the same way.
+Prawdopodobnie znasz już idee wtyczek. Wiele systemów obsługuje dodawanie niestandardowych wtyczek w celu dodania nowej funkcjonalności, a nawet zmodyfikowania podstawowych funkcji oprogramowania. Wtyczki Gatsby działają w ten sam sposób.
 
-Community members (like you!) can contribute plugins (small amounts of JavaScript code) that others can then use when building Gatsby sites.
+Członkowie społeczności (tacy jak Ty!) Mogą dodawać wtyczki (niewielkie ilości kodu JavaScript), z których inni mogą korzystać podczas tworzenia witryn Gatsby.
 
-> There are already hundreds of plugins! Explore the Gatsby [Plugin Library](/plugins/).
+> Istnieją już setki wtyczek! Poznaj [Bibliotekę wtyczek](/plugins/) Gatsby.
```

Comment:
Co powiesz na `Bibliotekę Wtyczek` z dużych liter, tak samo jak `Plugin Library`?
```suggestion
> Istnieją już setki wtyczek! Poznaj [Bibliotekę Wtyczek](/plugins/) Gatsby.
```

Reply:
Good catch :) Umknęło mi to.